src/app/Shared/LoadingOrError.component.tsx

```diff
@@ -1,14 +1,14 @@
 import React from "react";
-import { ErrorPageComponent } from "../ErrorPage/ErrorPage.component";
-import { LoadingProcessComponent } from "../../../Shared/LoadingProcess.component";
+import { ErrorPageComponent } from "../Main/Body/ErrorPage/ErrorPage.component";

```

Comment:
Reject: Shared components should never ever import something from your features or main module. So move the ErrorPage to shared or make the component to display on error parameterizable.

Reply:
Done. Commit "Make it simple"